### DISCUSSION TITLE
Extension Idea : Gmail Like Smart Compose

### DISCUSSION THREAD

**Discussion by varun-raj** [ueberdosis/tiptap#3679] [Feature Requests]
I was wondering if there is an extension already does this, if not it would be really great if there is plugin that can let us insert placeholder sort of text for autocomplete which gets filled when the user press tab on the keyboard. 

This could go well in applications trying to implement GPT3 based autocomplete

**Comment by samstephenson** [comment]
Did you ever get anywhere with this? I'm trying to implement something similar!
